src/Altinn.Platform/Altinn.Platform.Register/Register/Controllers/PartiesController.cs

```diff
@@ -72,5 +86,31 @@ public async Task<ActionResult> LookupPartyIdBySSNOrOrgNo([FromBody]string looku
 
             return NotFound();
         }
+
+        /// <summary>
+        /// Perform a lookup/search for a specific party by using one of the provided ids.
+        /// </summary>
+        /// <param name="partyLookup">The lookup criteria. One and only one of the properties must be a valid value.</param>
+        /// <returns>The identified party.</returns>
+        [ValidateModelState]
+        [HttpPost("lookup")]
+        [Consumes("application/json")]
+        [ProducesResponseType(400)]
+        [ProducesResponseType(404)]
+        [ProducesResponseType(200)]
+        [Produces("application/json")]
+        public async Task<ActionResult<Party>> PostPartyLookup([FromBody]PartyLookup partyLookup)
```

Comment:
This method does exactly the same as the other lookup method except for using POST instead of GEt, what is the intended use for this endpoint that couldn't be covered by the already implemented one?

Reply:
The primary purpose of this change was to introduce a POST version of the existing GET method(s).